packages/import-cost/src/webpack.js

```diff
@@ -27,6 +27,9 @@ function calcSize(packageInfo, callback) {
   const compiler = webpack({
     entry: entryPoint.name,
     plugins: [
+      new webpack.DefinePlugin({
+        'process.env.NODE_ENV': JSON.stringify('production')
```

Comment:
Was there any observable additional latency introduced by adding the `DefinePlugin` step in here?

Reply:
Not really, I think prod-mode slowdowns are due to uglification in most cases.

Reply:
That sounds about right. Cheers. Interested in @shahata's take on this PR.